affiliatewp-order-details-for-affiliates.php

```diff
@@ -290,11 +325,12 @@ public function has_2_1_7() {
 	}
 
 	/**
-	 * Add template folder to hold the customer table
+	 * Add template folder to hold the customer table.
```

Comment:
```suggestion
	 * Adds the template folder to hold the customer table.
```
Third-person singular verb in the DocBlock summary.